options/locale/locale_fa-IR.ini

```diff
@@ -328,7 +328,7 @@ reset_password=حساب خود را دوباره فعال کنید
 reset_password.title=%s، شما برای بازیابی حسابتان درخواست کرده‌اید
 reset_password.text=لطفاً روی پیوند زیر کلیک کنید تا حساب خود را در <b>%s</b> بازیابی کنید:
 register_success=ثبت‌نام با موفقیت انجام شد
-issue_assigned.pull=@%[1] به شما برای درخواست pull %[2] در ریپازیتوری %[3] محول شده.
+issue_assigned.pull=@%[1]s به شما برای درخواست pull %[2]s در ریپازیتوری %[3]s محول شده.
```

Comment:
I edited this string on Crowdin side, and it should be not wrong at least:

![image](https://github.com/go-gitea/gitea/assets/2114189/4c471b77-eb1b-480e-b64b-3b965afb60e8)